similaritymeasures/similaritymeasures.py

```diff
@@ -434,7 +434,7 @@ def curve_length_measure(exp_data, num_data):
     return np.sqrt(np.sum(r_sq))
 
 
-def frechet_dist(exp_data, num_data, p=2):
+def frechet_dist(exp_data, num_data, metric='euclidean'):
```

Comment:
Can you fix the flake8 errors here:
```
similaritymeasures/similaritymeasures.py:4:1: F401 'scipy.spatial.minkowski_distance' imported but unused
similaritymeasures/similaritymeasures.py:499:50: E251 unexpected spaces around keyword / parameter equals
similaritymeasures/similaritymeasures.py:499:52: E251 unexpected spaces around keyword / parameter equals
similaritymeasures/similaritymeasures.py:502:19: E231 missing whitespace after ','
similaritymeasures/similaritymeasures.py:504:39: E231 missing whitespace after ','
similaritymeasures/similaritymeasures.py:506:39: E231 missing whitespace after ','
similaritymeasures/similaritymeasures.py:510:31: E231 missing whitespace after ','
```

Reply:
Sincerely I did not run flake on my machine I don't quickly know how to do it, but I think I fixed everything. I would kindly ask you to rerun it and evaluate again if standards are met.